docs/snippets/about-revisions-garbage-collection.md

```diff
@@ -0,0 +1,7 @@
+<!-- Snippet used in the following topics:
+- docs/serving/revisions/revision-admin-config-options.md
+- docs/serving/revisions/revision-developer-config-options.md
+-->
+When Revisions of a Knative Service are inactive, they are automatically cleaned up and cluster resources are reclaimed after a set time period. This is known as *[garbage collection](https://kubernetes.io/docs/concepts/architecture/garbage-collection/){target=_blank}*.
```

Comment:
I would prefer the link here goes to the knative garbage collection page and not the kubernetes page explaining the concept of garbage collection (just a suggestion)

Reply:
So the reason for this is that this is the snippet that makes up the text on the Knative garbage collection pages, i.e. https://deploy-preview-5183--knative.netlify.app/docs/serving/revisions/revision-admin-config-options/ etc use this snippet

Otherwise the page would be pointing to itself, or we'd need to add a whole in-depth explanation here about what garbage collection is rather than just pointing to the conceptual k8s docs, and I think that's a bit outside the scope of this PR or our docs in general